DOc/docs/source/Options/data_quality.rst

```diff
@@ -213,81 +213,476 @@ These are they data quality rules that are applied on a dataset.
+Conditional unallowed text    Checks if values in the column with the given
+                              `value_column` contain a specific unallowed `text`.
+                              
+Conditional value             Checks whether the values in the given `value_column`
+                              match (one of) the expected value(s) for a given key
+                              in the `key_column`.
+                              
+Contains character            Checks how many times the values in the column with
+                              the given `column_name` contain a specific character.
+                              
+Formatting                    Checks whether or not the values in the column with
+                              the given `column_name` match the given `pattern`.
+                              
+Invalidity                    Checks whether or not the values in the column with
+                              the given `column_name` does not exist in the given
+                              list of `values`.
+                              
+Length                        Checks if the number of characters of the values in
+                              the column with the given `column_name` are equal to
+                              the `required_length`.
+                              
+Range                         Checks whether or not the values in the column with
+                              the given `column_name` are:
+                              - Greater than or equal to the given `lower_bound`.
+                              - Less than or equal to the given `upper_bound`.
+                              
+Starts with                   Checks whether or not the values in the column with
+                              the given `column_name` start with any of the given
+                              `prefixes`.
+                              
+Unallowed text                Checks if values in the column with the given
+                              `column_name` contain a specific unallowed `text`.
+                              
+Uniqueness                    Checks whether the values in the column with the given
+                              `column_name` are unique (duplicate value check).
+                              
+Validity                      Checks whether or not the values in the column with
+                              the given `column_name` exist in the given list of
+                              `values`.
+                              
+Cross-Column Validity         Checks whether or not the combination of values in the
+                              given `first_column_name` and `second_column_name`
+                              exist in the given list of valid `value_combinations`.
+
+							  
+============================  =======================================================
+
+    
+
+Data Quality Rules Examples With Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
```

Comment:
I think it is called bijection https://en.wiktionary.org/wiki/bijection
please adjust in the complete document; adjective bijective